.github/workflows/conformance-multi-pool.yaml

```diff
@@ -195,3 +195,17 @@ jobs:
           name: cilium-sysdump-out.zip
           path: cilium-sysdump-*.zip
           retention-days: 5
+
+      - name: Upload JUnits [junit]
+        if: ${{ always() }}
+        uses: actions/upload-artifact@0b7f8abb1508181956e8e162db84b466c27e18ce # v3.1.2
+        with:
+          name: cilium-junits
+          path: cilium-junits/*.xml
+          retention-days: 5
```

Comment:
I was just wondering if 5 days are required. It looks like other actions use 2 days.